src/frontend/app/features/endpoints/connect-endpoint-dialog/connect-endpoint-dialog.component.ts

```diff
@@ -2,6 +2,7 @@ import { Component, Inject, OnDestroy } from '@angular/core';
 import { FormBuilder, Validators } from '@angular/forms';
 import { MAT_DIALOG_DATA, MatDialogRef, MatSnackBar } from '@angular/material';
 import { Store } from '@ngrx/store';
+import { filter, map } from 'rxjs/operators';
 import { Observable } from 'rxjs/Rx';
```

Comment:
We seem to be using different imports: Its either `rxjs/Observable` or `rxjs/Rx`